Classes/OldModel/Event.php

```diff
@@ -593,14 +620,30 @@ protected function getPlaceWithDetailsRaw(): string
      */
     protected function getPlacesAsArray(): array
     {
-        return \Tx_Oelib_Db::selectMultiple(
-            'title, address, zip, city, country, homepage, directions',
-            'tx_seminars_sites, tx_seminars_seminars_place_mm',
-            'uid_local = ' . $this->getUid() . ' AND uid = uid_foreign' .
-            \Tx_Oelib_Db::enableFields('tx_seminars_sites'),
-            '',
-            'sorting ASC'
-        );
+        $queryBuilder = self::getQueryBuilderForTable('tx_seminars_sites');
+
+        return $queryBuilder
+            ->select('uid', 'title', 'address', 'zip', 'city', 'country', 'homepage', 'directions')
+            ->from('tx_seminars_sites')
+            ->from('tx_seminars_seminars_place_mm')
+            ->leftJoin(
+                'tx_seminars_sites',
+                'tx_seminars_seminars_place_mm',
+                'mm',
+                $queryBuilder->expr()->eq(
+                    'tx_seminars_sites.uid',
+                    $queryBuilder->quoteIdentifier('mm.uid_foreign')
+                )
+            )
+            ->where(
+                $queryBuilder->expr()->eq(
+                    'tx_seminars_seminars_place_mm.uid_local',
+                    $queryBuilder->createNamedParameter($this->getUid(), \PDO::PARAM_INT)
+                )
+            )
+            ->orderBy('tx_seminars_seminars_place_mm.sorting', 'ASC')
```

Comment:
The sorting should be numeric, not ASC.

Reply:
> The second, optional argument of both methods specifies the sorting order. The two allowed values are 'ASC' and 'DESC' where 'ASC' is default and can be omited.

https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Database/QueryBuilder/Index.html#orderby-and-addorderby

Reply:
Oh, you're completely right. I got confused. Sorry.